modules/p2/ding_entity_rating/js/ding_entity_rating_ajax.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
+ * Ajax rating field formatter behavior scripts.
+ */
+
+(function ($) {
+  "use strict";
+
+  // Drupal behavior.
+  Drupal.behaviors.ding_entity_rating_ajax = {
+    attach: function (context) {
+
+      var rating_ids = [];
+      $('.ding-entity-rating-display-ajax', context).once(function () {
+        rating_ids.push($(this).attr('data-ding-entity-display-ajax-rating-id'));
```

Comment:
No, not if i read the documentation at jquery correctly, but it might be that is just can't be used to update the data value. But I ended up during it this way as that was how the module already did it.